upup/models/cloudup/resources/addons/cluster-autoscaler.addons.k8s.io/k8s-1.15.yaml.template

```diff
@@ -257,8 +257,34 @@ spec:
   selector:
     app.kubernetes.io/name: "cluster-autoscaler"
   type: "ClusterIP"
+
+{{- if WithDefaultBool .ServiceMonitor.Enabled false }}
 ---
+# Source: cluster-autoscaler/templates/service-monitor.yaml
+apiVersion: monitoring.coreos.com/v1
```

Comment:
Oh - `monitoring.coreos.com` a surprising api group to see.  Obviously coreos/redhat/ibm were/are very much part of the kubernetes project, but this does mean that users will have to install the prometheus operator I guess (if they enable this option), and that operator isn't under CNCF I believe - though it is Apache 2.  Honestly I'm just not sure how we should handle that - I don't think we have a simple "yes/no" policy ....